src/components/offerStatus/ReasonAndTypeForm.js

```diff
@@ -0,0 +1,102 @@
+import PropTypes from 'prop-types';
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Alert, AlertVariants } from '../publiq-ui/Alert';
+import { RadioButtonGroup } from '../publiq-ui/RadioButtonGroup';
+import { getStackProps, Stack } from '../publiq-ui/Stack';
+import { Text } from '../publiq-ui/Text';
+import { TextAreaWithLabel } from '../publiq-ui/TextAreaWithLabel';
+import { getValueFromTheme } from '../publiq-ui/theme';
+import { MaxLengthReason, OfferStatus, OfferType } from './constants';
+
+const getValue = getValueFromTheme('statusPage');
+
+const ReasonAndTypeForm = ({
+  offerType,
+  statusType,
+  statusReason,
+  onChangeStatusType,
+  onInputStatusReason,
+  ...props
+}) => {
+  const { t } = useTranslation();
+
+  const radioButtonItems = useMemo(
+    () => [
+      {
+        label:
+          offerType === OfferType.PLACE
+            ? t('offerStatus.status.open')
+            : t('offerStatus.status.scheduled'),
```

Comment:
You could remove these `OfferType.PLACE ? t('...') : t('offerStatus.status.scheduled')` by using translation keys like:

```
{
    offerStatus: {
        event: {
            available: "Gaat door",
            temporarilyUnavailable: "Uitgesteld",
            unavailable: "Geannuleerd"
        },
        place: {
            available: "Open",
            temporarilyUnavailable: "Tijdelijk gesloten",
            unavailable: "Permanent gesloten"
        }
    }
}
```

And then calling it like `t('offerStatus.' + offerType + '.available')`